src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -128,25 +131,35 @@ const formConfig = {
     formSignature: {
       title: 'Signer information',
       pages: {
-        formSignature: {
-          path: 'form-signature',
+          formSignature: {
+          path: UPLOADS_COMPLETE_PATH,
+          // initialData: mockdata.data,
           title: 'Form signature',
           ...formSignatureSchema,
         },
+        signerEmail: {
+          path: 'signer-email',
+          title: 'Your email Address',
+          ...applicationEmailSchema,
+        },
       },
     },
     applicantInformation: {
       title: 'Beneficiary information',
       pages: {
         applicantNameDob: {
-          // initialData: mockdata.data,
           path: 'applicant-info',
           title: formData =>
             `${
               formData.certifierRole === 'applicant' ? 'Your' : 'Beneficiary’s'
             } name`,
           ...applicantNameDobSchema,
         },
+      },
+    },
+    applicantInformation: {
+      title: 'Beneficiary information',
+      pages: {
```

Comment:
This looks like we may have two duplicate chapter headers - could be result from a merge conflict?